### PR TITLE
fix pep8

### PR DESCRIPTION
**Pull request by nden**
This contains only pep8 changes and general clean up.
`countExtn` is now imported from `stsci_tools.fileutil`.

### CODE REVIEW

File: drizzlepac/hlautils/astrometric_utils.py [hunk 1]
```diff
@@ -884,31 +874,33 @@ def read_hlet_wcs(filename, ext):
     """
     hstwcs = wcsutil.HSTWCS(filename, ext=ext)
     if hstwcs.naxis1 is None:
```
Comment (mcara) [inline drizzlepac/hlautils/astrometric_utils.py]: @nden Is there any circumstance under which `naxis` could be `None` in a WCS?

Comment (nden) [reply drizzlepac/hlautils/astrometric_utils.py]: Only if you create a default WCS without passing a data file.
Unless `filename` above can be `None`, `naxis1/2` should always be populated.

Comment (mcara) [reply drizzlepac/hlautils/astrometric_utils.py]: @nden Thanks! I was just curious (+wanted to e sure)

Comment (mcara) [reply drizzlepac/hlautils/astrometric_utils.py]: @nden Then maybe this `if` can be removed?

Comment (nden) [reply drizzlepac/hlautils/astrometric_utils.py]: This PR is simply PEP8 changes. I'll try to review the code today and will include it there.

**Pull request merged**